### PR TITLE
Change: Clarify what effect town interactions have

### PR DESCRIPTION
**Pull request by Eddi-z**
## Motivation / Problem

It's never really explained anywhere what e.g. a statue actually does.

## Description

This adds a short description of the gameplay effects of several town interactions.

## Checklist for review

Some things are not automated, and forgotten often. This list is a reminder for the reviewers.
* The bug fix is important enough to be backported? (label: 'backport requested')
* This PR affects the save game format? (label 'savegame upgrade')
* This PR affects the GS/AI API? (label 'needs review: Script API')
    * ai_changelog.hpp, gs_changelog.hpp need updating.
    * The compatibility wrappers (compat_*.nut) need updating.
* This PR affects the NewGRF API? (label 'needs review: NewGRF')
    * newgrf_debug_data.h may need updating.
    * [PR must be added to API tracker](https://wiki.openttd.org/en/Development/NewGRF/Specification%20Status)

### CODE REVIEW

File: src/lang/english.txt [hunk 1]
```diff
@@ -3171,13 +3171,13 @@ STR_LOCAL_AUTHORITY_ACTION_NEW_BUILDINGS                        :Fund new buildi
...
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_ROAD_RECONSTRUCTION          :{YELLOW}Fund the reconstruction of the urban road network.{}Causes considerable disruption to road traffic for up to 6 months.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_STATUE_OF_COMPANY            :{YELLOW}Build a statue in honour of your company.{}Permanently boosts station rating in this town.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_NEW_BUILDINGS                :{YELLOW}Fund the construction of new commercial buildings in the town.{}Temporarily boosts town growth in this town.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_EXCLUSIVE_TRANSPORT          :{YELLOW}Buy 1 year's exclusive transport rights in town.{}Town authority will not allow passengers and cargo to use your competitor company's stations.{}Cost: {CURRENCY_LONG}
 STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_BRIBE                        :{YELLOW}Bribe the local authority to increase your rating, at the risk of a severe penalty if caught.{}Cost: {CURRENCY_LONG}
```
Comment (James103) [inline src/lang/english.txt:3181]: Maybe clarify what happens if you do get caught? The actual effect is:
* Your local authority rating drops to Poor.
* Your station ratings in the town drop to 0.
* You can't execute any other local authority actions there for another 6 months.

If you don't get caught, it should be clarified that your local authority rating with the town can't go above "Excellent" with this method.

Comment (Eddi-z) [reply src/lang/english.txt:3181]: i'm not sure it's a good idea to be this elaborate.

File: src/lang/english.txt [hunk 2]
```diff
@@ -3171,13 +3171,13 @@ STR_LOCAL_AUTHORITY_ACTION_NEW_BUILDINGS                        :Fund new buildi
...
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_MEDIUM_ADVERTISING           :{YELLOW}Initiate a medium local advertising campaign, to attract more passengers and cargo to your transport services.{}Temporarily boosts station rating in a medium radius around the town centre.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_LARGE_ADVERTISING            :{YELLOW}Initiate a large local advertising campaign, to attract more passengers and cargo to your transport services.{}Temporarily boosts station rating in a large radius around the town centre.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_ROAD_RECONSTRUCTION          :{YELLOW}Fund the reconstruction of the urban road network.{}Causes considerable disruption to road traffic for up to 6 months.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_STATUE_OF_COMPANY            :{YELLOW}Build a statue in honour of your company.{}Permanently boosts station rating in this town.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_NEW_BUILDINGS                :{YELLOW}Fund the construction of new commercial buildings in the town.{}Temporarily boosts town growth in this town.{}Cost: {CURRENCY_LONG}
```
Comment (2TallTyler) [inline src/lang/english.txt]: I would suggest removing the mention of "commercial" buildings. To me, this seems to suggest that it forces towns to build commercial buildings which accept Goods, but in reality it just increases growth rate.

File: src/lang/english.txt (hunk 1 above)
Comment (James103) [reply src/lang/english.txt:3181]: That was just me listing all of the possible effects. It's not necessary to actually include all of them in the description.

File: src/lang/english.txt (hunk 2 above)
Comment (Eddi-z) [reply src/lang/english.txt]: That sounds like a good idea.

File: src/lang/english.txt (hunk 1 above)
Comment (Eddi-z) [reply src/lang/english.txt:3181]: Then i don't understand what exactly you want changed.

File: src/lang/english.txt (hunk 2 above)
Comment (mattkimber) [reply src/lang/english.txt]: Does fund buildings still have the feature where it temporarily increases the size of the town zones? I think the original intention of that was to promote more commercial buildings to be built (due to a larger central zone) but with this mechanism not reliably producing new commercial buildings and different town sets introducing their own building selection rules I think removing "commercial" is sensible.

Comment (Eddi-z) [reply src/lang/english.txt]: I believe it still does that, which you can see by a bunch of pavements suddenly appearing in the town.

File: src/lang/english.txt [hunk 3]
```diff
@@ -3171,13 +3171,13 @@ STR_LOCAL_AUTHORITY_ACTION_NEW_BUILDINGS                        :Fund new buildi
...
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_LARGE_ADVERTISING            :{YELLOW}Initiate a large local advertising campaign, to attract more passengers and cargo to your transport services.{}Temporarily boosts station rating in a large radius around the town centre.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_ROAD_RECONSTRUCTION          :{YELLOW}Fund the reconstruction of the urban road network.{}Causes considerable disruption to road traffic for up to 6 months.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_STATUE_OF_COMPANY            :{YELLOW}Build a statue in honour of your company.{}Permanently boosts station rating in this town.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_NEW_BUILDINGS                :{YELLOW}Fund the construction of new buildings in the town.{}Temporarily boosts town growth in this town.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_EXCLUSIVE_TRANSPORT          :{YELLOW}Buy 1 year's exclusive transport rights in town.{}Town authority will not allow passengers and cargo to use your competitor company's stations.{}Cost: {CURRENCY_LONG}
```
Comment (LordAro) [inline src/lang/english.txt]:
```suggestion
STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_EXCLUSIVE_TRANSPORT          :{YELLOW}Buy 1 year's exclusive transport rights in town.{}Town authority will not allow passengers and cargo to use your competitor's stations.{}Cost: {CURRENCY_LONG}
```

File: src/lang/english.txt [hunk 4]
```diff
@@ -3171,13 +3171,13 @@ STR_LOCAL_AUTHORITY_ACTION_NEW_BUILDINGS                        :Fund new buildi
...
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_MEDIUM_ADVERTISING           :{YELLOW}Initiate a medium local advertising campaign, to attract more passengers and cargo to your transport services.{}Temporarily boosts station rating in a medium radius around the town centre.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_LARGE_ADVERTISING            :{YELLOW}Initiate a large local advertising campaign, to attract more passengers and cargo to your transport services.{}Temporarily boosts station rating in a large radius around the town centre.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_ROAD_RECONSTRUCTION          :{YELLOW}Fund the reconstruction of the urban road network.{}Causes considerable disruption to road traffic for up to 6 months.{}Cost: {CURRENCY_LONG}
+STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_STATUE_OF_COMPANY            :{YELLOW}Build a statue in honour of your company.{}Permanently boosts station rating in this town.{}Cost: {CURRENCY_LONG}
```
Comment (LordAro) [inline src/lang/english.txt]:
```suggestion
STR_LOCAL_AUTHORITY_ACTION_TOOLTIP_STATUE_OF_COMPANY            :{YELLOW}Build a statue in honour of your company.{}Provides a permanent boost to station rating in this town.{}Cost: {CURRENCY_LONG}
```
?
similar change to the temporary boosts

**Review by LordAro — APPROVED** [review]

**Pull request merged**